src/spikeinterface/core/template_tools.py

```diff
@@ -2,20 +2,32 @@
 import numpy as np
 import warnings
 
+from .template import Templates
+from .waveform_extractor import WaveformExtractor
 from .sparsity import compute_sparsity, _sparsity_doc
 from .recording_tools import get_channel_distances, get_noise_levels
 
 
+def _get_dense_templates_array(templates_or_waveform_extractor):
+    if isinstance(templates_or_waveform_extractor, Templates):
+        templates_array = templates_or_waveform_extractor.get_dense_templates()
+    elif isinstance(templates_or_waveform_extractor, WaveformExtractor):
+        templates_array = templates_or_waveform_extractor.get_all_templates(mode="average")
+    else:
+        raise ValueError("templates_or_waveform_extractor should be Templates or WaveformExtractor")
```

Comment:
```suggestion
        raise TypeError(f"templates_or_waveform_extractor should be Templates or WaveformExtractor not of type={type(templates_or_waveform_extractor)}")
```

This is a `TypeError` no?